app/src/main/java/com/example/scoutconcordia/Activities/MapsActivity.java

```diff
@@ -1195,6 +1208,26 @@ private void drawDirectionsPath(LatLng origin, LatLng dest){
         catch(Throwable t){
             Log.d(TAG, t.getMessage());
         }
+        int estimatedTime = new BigDecimal(time/60).setScale(0, RoundingMode.HALF_UP).intValue();
+        if(mode.toString().equals("driving") || mode.toString().equals("walking") || mode.toString().equals("transit")){
+//            Toast.makeText(MapsActivity.this, "estimated time in "+ estimatedTime + "mins",  Toast.LENGTH_LONG).show();
```

Comment:
Will we need this in the future? If not we can remove it

Reply:
we need these code to round up integers, for example if the time 0.1mins after divide by 60 then it will round up to 1 mins instead of 0 mins.